src/services/etherspot.js

```diff
@@ -292,7 +292,7 @@ export class EtherspotService {
 
   reserveEnsName(username: string): Promise<?ENSNode> {
     const fullEnsName = getEnsName(username);
-    return this.sdk.getAccountTokenListTokens({ name: fullEnsName }).catch((error) => {
+    return this.sdk.reserveENSName({ name: fullEnsName }).catch((error) => {
```

Comment:
This is silly :)

Reply:
Copy pasta?

Reply:
😂

Reply:
I noticed that during last week this is second time this happens 👀  hot weather? rest? 🌴